lib/streamlit/DeltaGenerator.py

```diff
@@ -2422,6 +2421,98 @@ def deck_gl_chart(self, element, spec=None, **kwargs):
 
         deck_gl.marshall(element.deck_gl_chart, spec, **kwargs)
 
+    @_with_element
+    def deck_json_chart(self, element, pydeck_obj=None):
+        """Draw a map chart using the Deck.GL library.
+
+        This API closely follows Deck.GL's JavaScript API
+        (https://deck.gl/#/documentation), with a few small adaptations and
+        some syntax sugar.
+
+        Parameters
+        ----------
+
+        pydeck_obj : pydeck intance or None (https://deckgl.readthedocs.io/en/latest/index.html)
+
+        Example
+        -------
+
+        >>> import pandas as pd
+        >>> import pydeck as pdk
+        >>> import streamlit as st
+
+        >>> DATA_URL = "https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/geojson/vancouver-blocks.json"
+        >>> LAND_COVER = [[[-123.0, 49.196], [-123.0, 49.324], [-123.306, 49.324], [-123.306, 49.196]]]
+        
+        >>> INITIAL_VIEW_STATE = pdk.ViewState(
+        ...    latitude=49.254,
+        ...    longitude=-123.13,
+        ...    zoom=11,
+        ...    max_zoom=16,
+        ...    pitch=45,
+        ...    bearing=0)
+
+        >>> polygon = pdk.Layer(
+        ...    'PolygonLayer',
+        ...    LAND_COVER,
+        ...    stroked=False,
+        ...    # processes the data as a flat longitude-latitude pair
+        ...    get_polygon='-',
+        ...    get_fill_color=[0, 0, 0, 20])
+
+        >>> geojson = pdk.Layer(
+        ...    'GeoJsonLayer',
+        ...    DATA_URL,
+        ...    opacity=0.8,
+        ...    stroked=False,
+        ...    filled=True,
+        ...    extruded=True,
+        ...    wireframe=True,
+        ...    get_elevation='properties.valuePerSqm / 20',
+        ...    get_fill_color='[255, 255, properties.growth * 255]',
+        ...    get_line_color=[255, 255, 255],
+        ...    pickable=True)
+
+        >>> r = pdk.Deck(layers=[polygon, geojson],initial_view_state=INITIAL_VIEW_STATE)
+        >>> st.write(r)
+
```

Comment:
This is the start of a whole new example, right? So add some text here to let the user know:

```
        >>> r = pdk.Deck(layers=[polygon, geojson],initial_view_state=INITIAL_VIEW_STATE)
        >>> st.write(r)

        And here is an example of placing icons on a map with an `IconLayer`:

        >>> icon_data = ...
```